packages/create-turbo/__tests__/cli.test.ts

```diff
@@ -38,70 +40,113 @@ describe("create-turbo cli", () => {
 
   afterAll(() => {
     jest.setTimeout(DEFAULT_JEST_TIMEOUT);
+    execSync("corepack enable", { stdio: "ignore" });
+  });
+
+  afterEach(() => {
     if (fs.existsSync(testDir)) {
       fs.rmSync(testDir, { recursive: true });
     }
   });
 
-  it("guides the user through the process", (done) => {
-    let cli = spawn("node", [createTurbo, "--no-install"], {});
-    let promptCount = 0;
-    let previousPrompt: string;
-    const messages: string[] = [];
-    cli.stdout.on("data", async (data) => {
-      let prompt = cleanPrompt(data);
+  it("default: guides the user through the process", async () => {
+    const cli = spawn("node", [createTurbo, "--no-install"], {});
```

Comment:
Our most-recent issue with this only showed up during the `install` phase. I'd do a test that actually does `install` for each of our package managers.